.goreleaser.yml

```diff
@@ -28,6 +28,26 @@ brews:
     test: |
       system "#{bin}/dagger version"
 
+# https://goreleaser.com/customization/nix/
+nix:
+  - name: dagger
+    repository:
+      owner: dagger
+      name: nix
+    commit_author:
+      name: dagger-bot
+      email: noreply@dagger.io
+    path: pkgs/dagger/default.nix
+    url_template: "https://{{ .Env.ARTEFACTS_FQDN }}/dagger/releases/{{ .Version }}/{{ .ArtifactName }}"
+    homepage: "https://github.com/dagger/dagger"
```

Comment:
Pointing to the website sounds better to me, but I copied this from the Homebrew section.

Reply:
Nice catch! Agree that we should set this to our website instead.